lib/twilio/audiohelper.ts

```diff
@@ -262,11 +268,16 @@ class AudioHelper extends EventEmitter {
   }
 
   /**
-   * Current state of the enabled sounds
+   * Destroy this AudioHelper instance
    * @private
    */
-  _getEnabledSounds(): Record<Device.ToggleableSound, boolean> {
-    return this._enabledSounds;
+  _destroy(): void {
```

Comment:
```suggestion
  private _destroy(): void {
```

Reply:
This is public internal. Meaning, customers cannot use it (not visible in public API), but internal classes can. See device.